python/nav/eventengine/alerts.py

```diff
@@ -143,7 +143,11 @@ def post(self, post_alert=True, set_state=True):
             if not alert:
                 alert = self.make_alert()
                 alert.history = history
-            export.exporter.export(alert)
+            try:
+                export.exporter.export(alert)
+            except Exception:
+                # we don't want to derail everything internally if external export fails
+                _logger.exception("Ignoring unhandled exception on alert export")
```

Comment:
Shouldn't you also log the actual exception? To be able to fix the exporting script later.

You could also change the exception to a warning.

Reply:
The logger method `exception()` logs an ERROR level message followed by a full traceback of the current exception.

Reply:
If you would insist on reducing the log level to WARNING, this would change to `_logger.warning("blah blah", exc_info=True)`

Reply:
As-is is good enough.